packages/rpc-api/src/__tests__/get-program-accounts-test.ts

```diff
@@ -2339,7 +2339,7 @@ describe('getProgramAccounts', () => {
                     filters: [
                         {
                             memcmp: {
-                                bytes: mint,
+                                bytes: mint as unknown as Base58EncodedBytes,
```

Comment:
Case in point.

Reply:
I suspect we shouldn't do this because of package dependencies, but it would kinda make sense for `Address` to be a branded `Base58EncodedBytes` instead of string. Then I don't think you'd need the cast to `unknown` here

Reply:
Oooooh, we should. I don't even think this would break anything more than usual, since if you have two different definitions of the _current_ `Address` type in the same project you're already in trouble.